kafka/channel/pkg/reconciler/controller/kafkachannel_test.go

```diff
@@ -390,6 +390,77 @@ func TestTopicExists(t *testing.T) {
 			dispatcherNamespace:      testNS,
 			dispatcherDeploymentName: testDispatcherDeploymentName,
 			dispatcherServiceName:    testDispatcherServiceName,
+			dispatcherImage:          testDispatcherImage,
+			kafkaConfig: &KafkaConfig{
+				Brokers: []string{brokerName},
+			},
+			kafkachannelLister: listers.GetKafkaChannelLister(),
+			// TODO fix
```

Comment:
wanna do an issue for that ?

Reply:
lol :) Copy and paste from an existing test, so not sure what the actual TODO fix is :) 
But, sure I'll create an issue :)

Reply:
https://github.com/knative/eventing-contrib/issues/915